armada-ths/ais
# Exhibitor personal view

Added dynamic exhibitor view. Fixed exhibitor image upload.

I made the thing fairly procedural, as in we can modify Exhibitor model and it will likely not require changes to my code, however should the need arise I left some hooks for special fields.

to test:
* make sure you are logged in on a user with exhibitors | exhibitor | view_exhibitors or a superuser
* make sure you have an Exhibitor (preferably, everything should work even without one)
* goto localhost:8000/fairs/2017/exhibitors
* click "edit view" button (or goto localhost:8000/fairs/2017/exhibitors/view)
* select the fields you like
* click save, and you should be redirected to exhibitors view
* make sure you see the fields you selected

If you want to you can also login on a different user and repeat the process to check that it is personal
